examples/recsys2020.ipynb

```diff
@@ -730,7 +1060,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 4. Train"
+    "## Training out model"
```

Comment:
This sentence reads weird to me - maybe either 'Training our model' or 'Training the model  instead' ?
```suggestion
    "## Training our model"
```

Reply:
Thanks, fixed